STYLEGUIDE.md

```diff
@@ -12,8 +12,7 @@ Our copy should be:
 ## General
 
 - Seek to make content available, and coherent, to all peoples.
-- Use Title Case for headers, buttons, features, and short lists.
-- Use sentence case for non-header text. When sentence case is used, full punctuation should be used.
+- Use sentence case for all text. When sentence case is used, full punctuation should be used.
```

Comment:
Full punctuation (periods, etc.) should not be used in many cases: buttons, labels, nav links, headlines, etc. I think you can just omit the "When sentence case is used, full punctuation should be used." sentence and it will be clear.

Reply:
BTW, the model for us to follow is GitHub. They have *very* consistent style in all their pages and materials, and they use sentence case. If in doubt about a rule, look at what GitHub does.